utils/mail.py

```diff
@@ -120,6 +127,7 @@ def send_mail(subject, email_body, user_to=None, email_to=None, email_from=None,
         for username, email in email_to:
             emails_logger.info('Email sent (%s)' % json.dumps({
                 'subject': subject,
+                'full_subject': full_subject,
```

Comment:
do we need both of these? In this case, full_subject contains both `[freesound]` and the extra if it's been set, but previously they `subject` line just contained the subject without the prefix. We could construct the subject without the prefix in order to log here, but then add it in order to send the email. What do you think?

Reply:
I'm interested in `subject` really, because it will be constant per email type (eg, all topic reply notifications will have the same `subject`, whereas `full_sibject` includes the topic as well). I think it is nice to log both because we might be interested in the rest. But we could log `extra_subject` instead!

Reply:
Done in 09377079f39a904c17a9e6dcfef699a7dfe02f8c